purchase_mass_mail/models/purchase_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def action_all_rfq_related_send(self):
+        template = self.env.ref("purchase.email_template_edi_purchase", False)
+        try:
+            compose_form = self.env.ref(
+                "mail.email_compose_message_wizard_form",
+                False,
+            )
+        except ValueError:
+            compose_form = False
```

Comment:
Look at line 36 below:

            "view_id": compose_form.id,

If the template is missing and this line is triggered and `compose_form` is really `False`, line 36 would fail because `False.id` fails with an `AttributeError`.

For that reason, I think you can leave the original exception and remove the try-catch block. That error is gonna be more obvious in the improbable case that it happens.